crates/typst/src/model/cite.rs

```diff
@@ -8,12 +8,11 @@ use crate::model::bibliography::Works;
 use crate::model::CslStyle;
 use crate::text::{Lang, Region, TextElem};
 
-/// Cite a work from the bibliography.
+/// 参考文献の引用。
 ///
-/// Before you starting citing, you need to add a [bibliography] somewhere in
-/// your document.
+/// 引用を始める前に、文書のどこかで [bibliography] を追加しておく必要があります。
```

Comment:
textlintではエラーとなってはいませんが、`jtf-style/3.1.1.全角文字と半角文字の間`に従ってスペースを削除したほうが良い気がします。

```suggestion
/// 引用を始める前に、文書のどこかで[bibliography]を追加しておく必要があります。
```

Reply:
ありがとうございます。修正して確認いたしました。